cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -494,13 +495,15 @@ def _find_duplicate_params(param_list):
     return duplicated_params
 
 
-def _find_overlapping_paths(paths_list):
+def _find_overlapping_paths(shared_paths_list, local_paths_list):
```

Comment:
Why do you need to add an extra parameter? 
If it is just to return the two overlapping paths I'd say we can avoid this extra complexity.

If I'm not missing something we just need to be sure that shared and local are not overlapping each other so we could just call:
```python
_find_overlapping_paths(shared_paths_list + local_paths_list)
````
If I'm missing something let's talk about this.

Reply:
local mount directories can overlap with each other. e.g. the mount dir on queue 1 is /scratch, the mount dir on queue 2 is /scratch/compute2